docs/modules/reference/pages/configuration/minion-confd/minion-confd.adoc

```diff
@@ -39,30 +39,48 @@ org.opennms.instance.id: "<instance id>"
 
 Writes specified configuration to `$\{MINION_HOME}/etc/instance-id.properties`.
 
-=== Kafka RPC
+=== Kafka
+
+When using Kafka instead of the embedded Active MQ, settings for Kafka IPC (Twin) will take precedence over any RPC and Sink specific settings.
```

Comment:
```suggestion
When using Kafka instead of the embedded Active MQ, settings for Kafka IPC (Twin) will take precedence over any RPC- and Sink-specific settings.
```